### PR TITLE
Initial setup for GAE hosting

### PR DESCRIPTION
**Pull request by hoangviet1993**
This PR moves files hosted by GAE into `/app` folder and adds an `app.yaml` to specify which dir/files are hosted.

PTAL @seanlip

### CODE REVIEW

**Comment by hoangviet1993** [comment]
Should I be adding the google-cloud-sdk into 3rd party installation here too?

**Comment by seanlip** [comment]
Probably yes, I think. Is that easy to do?

File: app.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+- url: /(.+)
```
Comment (seanlip) [inline app.yaml]: I am a bit confused about the conventions here. Isn't our entire website (so far) completely static? If so, let's just put the whole thing in a single static/ directory.

Comment (hoangviet1993) [reply app.yaml]: I did moved everything under `/app` to be uploaded to GCP. 
You can take a look at the new directory structure [here](https://github.com/oppia/foundation-website/tree/34afc76eadabc1ef8c76f3788c09a2150a571f65)

File: app.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  static_files: app/\1
```
Comment (seanlip) [inline app.yaml]: Can we change app/ to static/? Other than that I think all this looks good.

**Comment by seanlip** [comment]
Basically LGTM.

One last thought btw. Do you want to update the README file?

**Comment by hoangviet1993** [comment]
Sure! Updated README to now include GAE as well.

**Comment by seanlip** [comment]
OK. Thanks!

**Pull request merged**